bindings/protocols/mqtt/index.template.html

```diff
@@ -111,10 +118,9 @@ <h2>URL format</h2>
                 <li><code>{address}</code> is the IP address of the MQTT broker</li>
                 <li><code>{port}</code> is the port of the MQTT broker</li>
             </ul>
-            Notice that the URL does not contain any information regarding [[[#topicname]]] or [[[#topicfilter]]]. These two
-            parameters
-            must to be specified using their own parameter as described in [[[#form-terms]]]. This to avoid ambiguities
-            regarding the starting <code>/</code> character and the usage of <code>#</code> that can be confused with URL
+            Notice that the URL does not contain any information regarding [[[#topicname]]] or [[[#topicfilter]]]. 
+            These two parameters must to be specified using their own parameter as described in [[[#form-terms]]]. 
+            This to avoid ambiguities regarding the starting <code>/</code> character and the usage of <code>#</code> that can be confused with URL
```

Comment:
Would it be more precise to say "URL fragment *separator*" here?

```suggestion
            This is necessary to avoid ambiguities regarding the starting <code>/</code> character and the usage of <code>#</code> which can be confused with the URL
```

Reply:
Oh, I actually meant the "fragment character" in the following line.